cheatsheet.md

```diff
@@ -11,98 +11,104 @@ with possibly out-of-date explainers.
-the locked ancestors of the focused element will be activated
-* Can disabled with `skip-viewport-activation` or `skip-activation`.
+  the locked ancestors of the focused element will be unlocked. Note that this
+  only applies to `render-subtree: invisible` configuration. other configurations
+  pay no special attention to sequential focus navigation.
 
 ### Find-in-page
-* Find-in-page will find text even in locked subtrees, counting them in the total match.
-* Find-in-page won't activate *all* text that match - it will only activate one main/currently-selected match (see below).
-* When find-in-page navigates to an active match (currently-highlighted/selected match) because it's the first match or through find-next/find-prev navigation, the locked ancestors of the active match text will be activated.
-* Can be disabled with `skip-activation`.
+* Find-in-page will find text even in locked subtrees if the configuration
+  allows for it. The active or main match will cause all of its locked ancestors
+  to be unlocked. Tentative design decision: for `render-subtree: invisible
+  skip-viewport-activation` cases, the find-in-page algorithm, upon finding an
+  active match in a locked subtree, will issue a signal and
+  yield until script had a chance to react to the signal. At this time, the
+  algorithm will resume and only consider the active match as valid if the
+  subtree is now unlocked. Otherwise, it will proceed with looking for other
+  matches outside of the current locked root.
 
 ### Accessibility
-* Nodes in locked subtree (except those with `skip-activation`) are included in the AX tree but are marked as offscreen and don't have layout values, and thus are exposed to assistive technologies.
-* AX selections, focus, scrolling/navigation to nodes in locked subtrees will activate the locked ancestors.
-* Can be disabled with `skip-activation`.
-* There's still an active discussion on this, please see https://github.com/WICG/display-locking/issues/102 for more.
+* Subtrees in the `render-subtree: invisible` configuration are included in the
+  accessibility tree. Subtrees in other configurations are omitted. Note that
+  there may be other ways of bringing the content into view from accessibility
+  technology.  These algorithms should, for the most part, behave consistently
+  with non-accessibility ways of bringing content into view.
 
 ### Anchor link navigation:
-* When fragment link (ie url.html#elementid) navigation results in a navigation to an element in a locked subtree, its locked ancestors will be activated.
-* Can be disabled with `skip-activation`.
+* When fragment link (ie url.html#elementid or url.html#:~:text=foo) navigation
+  results in a navigation to an element in a locked subtree, its locked
+  ancestors will be unlocked. Tentative design: Similar situations as in
+  find-in-page cae should be considered. In particular, it may be prudent to
```

Comment:
Why say "prudent to" if it's a tentative decision anyway? Just say it should yield and then apply the scroll in a subsequent rendering task.